great_expectations/datasource/data_connector/data_connector.py

```diff
@@ -257,6 +257,7 @@ def self_check(self, pretty_print=True, max_examples=3):
                 f"\n\tUnmatched data_references ({min(len_unmatched_data_references, max_examples)} of {len_unmatched_data_references}):",
                 unmatched_data_references[:max_examples],
             )
+            print()
```

Comment:
Adding one more line to better-separate data connectors when printing to console

Reply:
Thoughts on adding the print to the previous line? I see no issues with your change but maybe we can squeeze in a `\n` in the prior print statement?

```
if pretty_print:
  print(
      f"\n\tUnmatched data_references ({min(len_unmatched_data_references, max_examples)} \
      of {len_unmatched_data_references}): {unmatched_data_references[:max_examples]}\n"
  )
```
Or maybe that's a little harder to understand? I'll trust your judgement here 😄